hr_period/README.rst

```diff
@@ -0,0 +1,95 @@
+HR Payroll Period
+=================
+
+Add the concept of payroll period.
+
+The objective is not to restrict or complicate the payroll workflow.
+It is meant to reduce the number of clicks related to date selections and avoid mistakes.
+
+Add the date of payment on the payslip and payslip batch. This date is automatically filled when selecting
+a period.
+
+Add the company field on the payslip batch.
+Add a sequence on the payslip batch name.
+
+
+Installation
+============
+
+Install the payroll of your localisation, then install this module.
+
+
+Configuration
+=============
+
+Create a fiscal year
+--------------------
+Go to: Human Resources -> Configuration -> Payroll -> Payoll Fiscal Year
+
```

Comment:
@pedrobaeza There's 2 models. The first is the payroll period (e.g. a month or a week or ..). The second is the Fiscal year, which is an object that helps create and manage periods over a year. Though, I wouldn't mind renaming Payroll Fiscal Year to Payroll Year.